_sass/content.scss

```
@@ -112,3 +112,57 @@
     font-weight: 500;
   }
 }
+
+banner {
```

```suggestion
banner {
  width: 100%
```

To be added with my previous suggestion